GirlsCode MK 1st Blog Post

edieblu opened the pull request.

edieblu committed:
team GirlsCode MK Introduction Blog Post

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
changed h3 formatting

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Final version of GirlsCode MK Blog

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

edieblu committed:
Update 2018-07-08-4Ps-of-GirlsCode-MK.md

v-thakkar commented on blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md:
```diff
@@ -0,0 +1,134 @@
+---
+title: Team GirlsCode MK - The 4 Ps of GirlsCode MK
```
Hi @edieblu ! Thanks for the PR. Please check the comments below.

v-thakkar commented on blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md:
```diff
@@ -0,0 +1,134 @@
+title: Team GirlsCode MK - The 4 Ps of GirlsCode MK
+layout: post
+created_at: Sun Jul 08 2018
+permalink: blog/_posts/2018-07-08-4ps-of-girlscode-mk
```
Here, permalink need to be same as file name. Please check the old blog posts for an example.

v-thakkar commented on blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md:
```diff
@@ -0,0 +1,134 @@
+created_at: Sun Jul 08 2018
+permalink: blog/_posts/2018-07-08-4ps-of-girlscode-mk
+author: Team GirlsCode MK
+twitter: GirlsCodeMK
```
The file is missing bunch of parameter declarations like 'current' and 'categories'. Please add them.

v-thakkar commented on blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md:
```diff
@@ -0,0 +1,134 @@
+
+## You can follow our journey here:
+
+__[@GirlsCodeMK](https://twitter.com/GirlsCodeMK)__
```
May be it would be good to write whose twitter accounts these are. You can have your names as a text [linking to twitter link] instead of twitter handles.

edieblu replied:
Thank you Vaishali :)

edieblu committed:
Updated blog following Vaishali's comments

edieblu committed:
updated

edieblu commented:
@nerdyvaishali Wohooo 👍

v-thakkar commented on Gemfile.lock:
```diff
@@ -125,9 +125,12 @@ PLATFORMS
 
 DEPENDENCIES
   geocoder
-  github-pages (= 84)
```
Hi @edieblu ! Usually we keep do change per PR. So, please send Gemfile changes in a separate PR if you wish to send it.

edieblu replied:
Hm, not sure how that got there @nerdyvaishali. Will fix it now :)

edieblu committed:
Update Gemfile.lock

edieblu committed:
Update Gemfile.lock

edieblu reviewed (commented):
Hopefully, it's okay now. Will try not to do the "git add ." again :)

v-thakkar committed:
Update the date and fix permalink

Commit:
Update date on GirlsCode MK blog post and formatting

Updated the date on the post and md file
Updated the date to the images
Minor changes to formatting
Rearranged the photos next to the titles
